tests/test_tk_docs_preview.py

```diff
@@ -30,7 +32,7 @@ def test_generate_doc_without_any_parameters():
     Make sure we can generate documentation from inside a repo without any arguments.
     """
     cwd = os.getcwd()
-    os.chdir(os.path.join(REPOS_ROOT, "tk-core"))
+    os.chdir(TK_FRAMEWORK_ROOT)
```

Comment:
It's better to run the test on the framework because it causes detection of the bundle and the core instead of just the core.